api/metrics-api-jaxrs/src/main/webapp/static/index.html

```diff
@@ -0,0 +1,42 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright (c) 2011, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html>
+<head>
+    <title>Hawkular Metrics</title>
+    <link rel="shortcut icon" href="metrics/static/favicon.ico" type="image/x-icon">
+    <link rel="StyleSheet" href="metrics/static/welcome.css" type="text/css">
+    <script src="metrics/static/status.js"></script>
+</head>
+
+<body>
+  <div class="content">
+    <div class="logo"/>
+      <img src="metrics/static/hawkular_logo.png" alt="Hawkular Logo"/>
```

Comment:
Le 06/07/2015 15:28, mwringe a écrit :
> It worked last time I checked it, are you not seeing the images or
> javascript working anymore?
>

No, I think it only works when you use the Docker image, because you 
load the content of this page from the root context with meta refresh.

But when you browse to /hawkular/metrics you can't see anything (no 
image, no status)